tests/main/snap-disconnect/test-snap-consumer.v2/meta/snap.yaml

```diff
@@ -2,4 +2,4 @@ name: test-snap-consumer
 version: 2
 plugs:
     other:
-        interface: dummy
+        interface: empty
```

Comment:
Just as a sanity check maybe it's useful to reach out to the store team to check if there aren't any snaps using that interface.

Reply:
As discussed on MM, review-tools complain about dummy iface so snap upload would be impossible.